appcues/src/main/java/com/appcues/statemachine/Transitions.kt

```diff
@@ -230,11 +230,11 @@ private fun errorTransition(experience: Experience, currentStepIndex: Int, messa
     )
 }
 
-private fun stepNotFoundErrorTransition(experience: Experience, groupIndex: Int, stepIndex: Int): Transition {
+private fun stepNotFoundErrorTransition(experience: Experience, groupIndex: Int): Transition {
```

Comment:
small adjustment to this error since its only being called passing step group and step 0,0